util/site-dashboard/dashboard.js

```diff
@@ -146,18 +146,23 @@ class DashboardTable {
 }
 
 class BlockLevelResults {
-  constructor(json, report_url) {
+  constructor(name = undefined, json, report_url) {
+    this.name = name;
     this.process_report(json);
     this.report_url = report_url;
   }
 
   process_report(report) {
-    this.block_name = report.block_name;
+    if (!(this.name)) {
+        this.block_name = report.block_name;
```

Comment:
Wrong indentation -- do we have auto formatter and checks?

Reply:
Oh yeah sorry I forgot the indentation.
Short answer is no and no for .js code at this moment.

Reply:
Maybe not worth auto checking JS if it requires bringing in Node.js

Reply:
👻 NodeJS 👻